build.sh

```diff
@@ -146,15 +146,27 @@ generate_event_logging_sources()
 # Event Logging Infrastructure
    __GeneratedIntermediate="$__IntermediatesDir/Generated"
    __GeneratedIntermediateEventProvider="$__GeneratedIntermediate/eventprovider_new"
+   __GeneratedIntermediateEventPipe="$__GeneratedIntermediate/eventpipe_new"
```

Comment:
I ran this change through the CI and got a build failure.  Can you double check this?  Log is at https://ci.dot.net/job/dotnet_coreclr/job/master/job/checked_windows_nt_bld_prtest/9133/consoleText.